gulpfile.babel.js

```diff
@@ -46,6 +49,16 @@ function bundle() {
         .pipe(gulp.dest('./'));
 }
 
+function compress(cb) {
+  pump([
+        gulp.src('compiled.js'),
+        uglify(),
+        gulp.dest('./')
```

Comment:
I wasn't sure if I should overwrite `compiled.js` or generate it in `dist/compiled.js` or `compiled.min.js`. 

Since we use the string `compiled.js` I didn't want to put `compiled.min.js` since we'd have changed the name everytime. Which one do you think would be the best for future developers and us?